modules/ROOT/pages/security/coding.adoc

```diff
@@ -0,0 +1,119 @@
+include::{partialsdir}/attributes.adoc[]
+
+= Coding an App to Use Device Security
+
+include::configuring-dev-env-security.adoc[leveloffset=1]
+
+== Setting up {device-security-service} service SDK
+
+This guide will help you to set up the {device-security-service} service SDK in your App.
+
+=== Importing the libraries
+
+[role="primary"]
+.Android
+
+****
+
+. Add `jcenter` to your project's *build.grade*:
++
+[source, groovy]
+----
+allprojects {
```

Comment:
that part is already explained in `build/site/aerogear/latest/configuring-dev-env.html`. Maybe just add that read as a prereq?

Reply:
I was also gonna suggest the same thing.

Reply:
Yes, I am editing it to look just like IDM